dashboard/config/application.rb

```diff
@@ -102,6 +102,10 @@ class Application < Rails::Application
       end
     end
 
+    config.after_initialize do
```

Comment:
why is this necessary?

Reply:
The fallbacks seem to be overwritten at some point after config initialization. 
I found this at http://pawelgoscicki.com/archives/2015/02/enabling-i18n-locale-fallbacks-in-rails/ (linked in the description).

Reply:
Is that also true for `config.i18n.fallbacks[:defaults] = ['en-US']` above?

Reply:
The default locale seems to work a bit differently for some reason. Even the [docs](https://github.com/ruby-i18n/i18n/wiki/Fallbacks) linked by the i18n rails gem has custom fallbacks modifying `I18n.fallbacks` instead of `config.i18n.fallback`.

If there's a better place to put this, I'm happy to move it there but I don't think it can be specified in `config.i18n`.

Reply:
Weird. This seems good, then, but would you add a comment explaining why we're doing it in this file and in the block? Possibly add one or both of those links